lib/fastlane/plugin/wpmreleasetoolkit/actions/common/create_release_action.rb

```diff
@@ -83,6 +83,12 @@ def self.available_options
                                        optional: true,
                                        default_value: false,
                                        is_string: false),
+          FastlaneCore::ConfigItem.new(key: :is_draft,
+                                       env_name: 'GHHELPER_CREATE_RELEASE_IS_DRAFT',
+                                       description: 'True if this is a draft release',
+                                       optional: true,
+                                       default_value: true,
+                                       is_string: false),
```

Comment:
💄  Nit: `is_string` is the legacy way of declaring a non-string parameter. We still have many `ConfigItem` that are using `is_string:` for legacy reasons, but we hope to get rid of them all to migrate to use the more modern `type:` attribute instead (see https://github.com/wordpress-mobile/release-toolkit/issues/278)

While it's outside of this PR to change all the existing `is_string` attributes to the more modern `type:` instead, it would be nice to at least use `type:` for any newly introduced `ConfigItem`:
```suggestion
                                       type: Boolean),
```